Specification.EntityFramework6/src/Ardalis.Specification.EntityFramework6/RepositoryBaseOfT.cs

```diff
@@ -10,8 +10,8 @@ namespace Ardalis.Specification.EntityFramework6;
 /// <inheritdoc/>
 public abstract class RepositoryBase<T> : IRepositoryBase<T> where T : class
 {
-    private readonly DbContext _dbContext;
-    private readonly ISpecificationEvaluator _specificationEvaluator;
+    protected readonly DbContext _dbContext;
```

Comment:
I think a property

`protected DbContext { get; private set; }`

combined with 

```
public RepositoryBase(DbContext dbContext)
{
    DbContext = dbContext;
}
```

is more appropriate then a protected member.

The same for `SpecificationEvaluator`

Reply:
Didn't want to change too much, but I see your point. If we let subclasses access these "fields" it will be safer to do "props" instead from a maintenance perspective. I've made that change and pushed